frontend/integration-tests/tests/performance.scenario.ts

```diff
@@ -33,18 +31,6 @@ const chunkedRoutes = OrderedMap<string, {section: string, name: string}>()
 
 describe('Performance test', () => {
 
-  it('checks bundle size using ResourceTiming API', async() => {
```

Comment:
Don't we still want this? It captures the actual chunks received by the browser.

Reply:
Is it working? I thought the bundle-analysis file always 0 and you asked me to remove :)

Reply:
Circling back on this... This is the test that wrote `bundle-analysis.txt` to the filesystem and always returned 0. I think having the full report is much more valuable regardless.

I haven't removed the tests for the various routes below.